habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/InstallDependenciesMojo.java

```diff
@@ -171,23 +171,18 @@ public void doExecute() throws MojoExecutionException, MojoFailureException {
     }
 
     private void setUpInProjectVirtualEnvironment(PoetryCommandHelper poetryHelper) throws MojoExecutionException {
-        String inProjectVirtualEnvironmentPath = HabushuUtil.getInProjectVirtualEnvironmentPath(getPoetryProjectBaseDir());
-        File venv = new File(inProjectVirtualEnvironmentPath);
-        if (this.useInProjectVirtualEnvironment) {
-            if (!venv.exists()) {
-                getLog().info("Configuring Poetry to use an in-project virtual environment...");
-                configureVirtualEnvironmentsInProject(true);
-            }
-        } else {
-            List<String> arguments = new ArrayList<>();
-            arguments.add("config");
-            arguments.add("virtualenvs.in-project");
-            arguments.add("--local");
-            String currentInProjectSetting = poetryHelper.execute(arguments);
-
-            if (Boolean.TRUE.equals(Boolean.valueOf(currentInProjectSetting))) {
-                configureVirtualEnvironmentsInProject(false);
-            }
+        List<String> arguments = new ArrayList<>();
+        arguments.add("config");
+        arguments.add("virtualenvs.in-project");
+        arguments.add("--local");
+        String currentInProjectSetting = poetryHelper.execute(arguments);
```

Comment:
Q: This is just to ensure we've got the poetry.toml file sync'd in the case that the user has a pre-existing .venv folder, but the config hasn't been set in the toml file? (Perhaps in the case it was manually modified to remove the config?)

Reply:
Correct, see [comment below](https://github.com/TechnologyBrewery/habushu/pull/182#discussion_r1698948930)